### PR TITLE
Enhanced User Experience and Code: Improved Todo List Functionality and UI Styling

### PR DESCRIPTION
**Pull request by MuchaSsak**
I was 2 commits behind so I merged the changes and now everything's up to date and created a fresh pull request as you've asked
Now you can simply clone my repo to test the code locally
Is that okay now?

### CODE REVIEW

**Comment by MuchaSsak** [comment]
Oh yeah and I've now noticed that you renamed my previous PR title good job haha naming stuff is so hard for me sorry for that

File: .eslintrc.cjs [hunk 1]
```diff
@@ -16,5 +16,6 @@ module.exports = {
       'warn',
       { allowConstantExport: true },
     ],
+    "react/prop-types": "off"
```
Comment (aman-sinha-dev) [inline .eslintrc.cjs:19]: Why this?

Comment (MuchaSsak) [reply .eslintrc.cjs:19]: when i added props to newly added components (e.g. `handleClickTaskAdd` in `Form.jsx`), then there appeared an eslint error saying that these props have `any` type

now, i could use PropTypes and specify the types as requested, but the thing is that this is not really used anymore. if you want to set types strictly, you should probably use TypeScript for that.

but that's just an opinion though

Comment (MuchaSsak) [reply .eslintrc.cjs:19]: that's weird. i don't see the error anymore when i remove this property. i also don't recall exactly what was the message.
i guess i just figured out to disable PropTypes all together 🤷‍♂️

**Review by aman-sinha-dev — CHANGES_REQUESTED** [review]
"react/prop-types": "off"
 
 remove this line.. rest everything is okk.....

**Review by aman-sinha-dev — APPROVED** [review]

**Pull request merged**